.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Unit tests
```

Comment:
@ReeceStevens this looks great! Nice work!

If you feel up to it, it could be nice to set up a GitHub action that publishes to pypi. See

https://github.com/innolitics/rdm/blob/main/.github/workflows/publish.yml

for an example file. You'll also need to create repository secretes for the Pypi credentials. See this link for details about how to get these:

https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#saving-credentials-on-github